packages/editor/src/components/provider/index.js

```diff
@@ -208,11 +216,18 @@ export default compose( [
 			setupEditor,
 			updatePostLock,
 			createWarningNotice,
-			resetEditorBlocks,
+			resetEditorBlocks( blocks, selectionStart, selectionEnd ) {
```

Comment:
To the above point, I wonder if any place we reference the selection, if it can be expressed as a single array tuple value, rather than as two separate values?

Reply:
What's better about `change( [ start, end ] )` versus `change( start, end )`?

Reply:
If you mean storing it as an array in the reducers, I also wonder how it would be better than a flat structure `selectionStart: {}` or nested structure `selection: { start: {} }`. If you'd have to write a selector, it would look something like `selection[ 0 ]`, which reads worse to me than both the other ways.

Reply:
To me, it's less about the structure itself, more thinking about ways to remove some of the redundancy of common logic in what we have now with `selectionStart`, `selectionEnd`.